bindgen/spec.yml

```diff
@@ -1208,13 +1209,13 @@ classes:
   SyncUser:
     cppName: SyncUser
     sharedPtrWrapped: SharedSyncUser
+    properties:
+      is_logged_in: bool
 
   User:
-    base: SyncUser
     cppName: app::User
     sharedPtrWrapped: SharedUser
     properties:
-      is_logged_in: bool
```

Comment:
I moved this to the `SyncUser` as this is where it's actually implemented:

https://github.com/realm/realm-core/blob/a45922fa249a9cf58c1452588ea3b3075e45eae2/src/realm/object-store/sync/sync_user.hpp#L47-L50